### PR TITLE
DAP-941: Worker and copyWorker functions

### PR DESCRIPTION
**Pull request by BradyMitch**
## 🎯 Summary

  
[DAP-941](https://citz-cirmo.atlassian.net/browse/DAP-941)

- Updated README ( Please Read ;) )
- Set up worker pool and environment for workers to work.
- Added copy worker (purpose explained in comment in file).

Tested in development and executable.

## 🔰 Checklist

- [x] I have read and agree with the following checklist.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.
> - My changes generate no new warnings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,23 +8,57 @@ The DATS project will be transferring inactive Full Retention (FR) government di
...
 
-2. Run `npm run up` to start the API, MongoDB, and RabbitMQ services.
+2. **Start Services**: Run `npm run up` to start the API, MongoDB, and RabbitMQ services.
 
-3. Change directory to `desktop` and run `npm run dev` to launch the desktop app.
+3. **Desktop Directory**: Navigate to `desktop`.
+
+4. **Install Dependencies**: Run `npm install`.
+
+5. **Build App**: Run `npm run build` to enable worker functionality.
```
Comment (TaylorFries) [inline README.md:19]: add note on running as admin based on os?

File: README.md [hunk 2]
```diff
@@ -8,23 +8,57 @@ The DATS project will be transferring inactive Full Retention (FR) government di
...
+npm run build:win
+npm run build:mac
+npm run build:linux
+```
+
+3. **Uninstall Previous Version**: Remove existing installation (e.g., in `C:\Users\<username>\AppData\Local\Programs\desktop` on Windows).
+
+4. **Locate Build**: In VSCode, right-click `desktop/dist/` and select `Reveal in File Explorer`. Run the setup executable.
+
+5. **Test App Launch**: Ensure the desktop app opens correctly, then close it.
```
Comment (TaylorFries) [inline README.md:59]:
```suggestion
3. **Uninstall Previous Version**: Remove existing installation
    a. Windows: in `C:\Users\<username>\AppData\Local\Programs\desktop`.
    b. Mac: `/Users/<username>/Applications/desktop.app`.

4. **Locate Build**: In VSCode, right-click `desktop/dist/` and select `Reveal in Finder/File Explorer`. Run the setup executable.

5. **Test App Launch**: Ensure the desktop app opens correctly, then close it.
```

File: README.md [hunk 3]
```diff
@@ -8,23 +8,57 @@ The DATS project will be transferring inactive Full Retention (FR) government di
...
 
-By default when running in development using `npm run dev`, the default selection is `Local` and when running in the executable the default selection is `Prod`.
+6. **Run in Console**: Open Command Prompt and run `<file-location>/desktop.exe` (e.g., `C:\Users\<username>\AppData\Local\Programs\desktop\desktop.exe`). Execution may differ on Mac, use Terminal to run the executable.
```
Comment (TaylorFries) [inline README.md:61]:
```suggestion
6. **Run in Console**:
    a. Windows: Open Command Prompt and run `<file-location>/desktop.exe`
        e.g., `C:\Users\<username>\AppData\Local\Programs\desktop\desktop.exe`
    b. Mac: Open Terminal and run `open <file location>`
        e.g., `open /Applications/desktop.app`
```

File: README.md (hunk 1 above)
Comment (BradyMitch) [reply README.md:19]: Great idea!

Comment (BradyMitch) [reply README.md:19]: Actually now that I think about it I'm not sure if it's necessary. On Windows I can run `npm run build` without Admin privs, but not `npm run build:win`.

`npm run build` just creates the `out` directory and not the `dist`. Is it the same on Mac?

File: README.md [hunk 4]
```diff
@@ -8,23 +8,63 @@ The DATS project will be transferring inactive Full Retention (FR) government di
...
+4. **Locate Build**: In VSCode, right-click `desktop/dist/` and select `Reveal in Finder/File Explorer`. Run the setup executable.
+
+5. **Test App Launch**: Ensure the desktop app opens correctly, then close it.
 
-By default when running in development using `npm run dev`, the default selection is `Local` and when running in the executable the default selection is `Prod`.
+6. **Run in Console**:
+    a. Windows: Open Command Prompt and run `<file-location>/desktop.exe`
+        e.g., `C:\Users\<username>\AppData\Local\Programs\desktop\desktop.exe`
+    b. Mac: Open Terminal and run `open <file location>`
+        e.g., `open /Applications/desktop.app`
```
Comment (TaylorFries) [inline README.md:67]:
```suggestion
    b. Mac: Open Terminal and run `<file location>`
        e.g., `/Applications/desktop.app/Contents/MacOS`
```

File: .gitignore [hunk 5]
```diff
@@ -4,3 +4,4 @@ node_modules
 package-lock.json
 .env
 coverage
+desktop/resources/transfers
```
Comment (TaylorFries) [inline .gitignore:7]:
```suggestion
desktop/resources/transfers
.DS_Store
```

**Review by TaylorFries — APPROVED** [review]
Works really well and I totally understand workers

**Pull request merged**